playbooks/adhoc/create_pv/create_pv.yaml

```diff
@@ -20,7 +20,7 @@
   pre_tasks:
   - fail:
       msg: "This playbook requires {{item}} to be set."
-    when: "{{ item }} is not defined or {{ item }} == ''"
+    when: {{ item }} is not defined or {{ item }} == ''
```

Comment:
This is different from how it was fixed on master. Need to get rid of jinja delimiters here too.